index.html

```diff
@@ -0,0 +1,262 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header id="header">
+        <nav class="navbar">
+            <a href="">
+                <h2>My Logo</h2>
+            </a>
+            <img src="images/hamburger.svg" alt="menu button">
+        </nav>
+    </header>
+    <section id="headline">
+        <h1>I’m Thinus Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
+        <ul class="social-links">
+            <li>
+                <a href="#"><img src="images/twitter.svg" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/linkedinicon.svg" alt="LinkedIn"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/github.svg" alt="Github"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector (1).svg" alt="Angel List"></a>
+            </li>
+        </ul>
+    </section>
+
+    <section id="works">
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio.svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Tonic</p>
+                    <section id="frame2">
```

Comment:
- [ ] Please note that you should have only one h1 in the whole of your document; kindly change this to any other heading tag(from h2 to h5). It is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.